x-pack/plugins/apm/public/components/shared/ImpactBar/index.tsx

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiProgress } from '@elastic/eui';
+import React from 'react';
+
+// TODO: extend from EUI's EuiProgress prop interface
+interface Props extends StringMap<any> {
```

Comment:
Looks like this already exists https://github.com/elastic/eui/pull/326

Reply:
Yeah it's not clear to me how to access these.